src/openforms/emails/tests/test_migrations.py

```diff
@@ -0,0 +1,75 @@
+from openforms.utils.tests.test_migrations import TestMigrations
+
+
+class ConfirmationEmailTemplatesCosignTest(TestMigrations):
```

Comment:
can you add a case where the space is missing after the percent sign? `{%summary%}` is valid template code, but won't be picked up by your regex